File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/MonthlyScheduledWorkPoolUser.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class MonthlyScheduledWorkPoolUser
    {
        public int MonthlyScheduledWorkPoolUserId { get; set; }
        public WorkPoolEnum WorkPool { get; set; }
        public System.DateTime StartDate { get; set; }
        public System.DateTime EndDate { get; set; }
        public int AssignedByUserId { get; set; }
        public int AssignedToUserId { get; set; }
        public bool IsDeleted { get; set; }
        public string CreatedBy { get; set; }
        public System.DateTime CreatedDate { get; set; }
        public string ModifiedBy { get; set; }
        public System.DateTime ModifiedDate { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/personEvent/monthly-scheduled-work-pool-user.ts
import { WorkPoolEnum } from 'projects/shared-models-lib/src/lib/enums/work-pool-enum';
import { User } from 'projects/shared-models-lib/src/lib/security/user';

export class MonthlyScheduledWorkPoolUser {
    monthlyScheduledWorkPoolUserId: number;
    workPool: WorkPoolEnum;
    startDate: Date;
    endDate: Date;
    assignedByUserId: number;
    assignedToUserId: number;
    isDeleted: boolean;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;
}